.github/workflows/go.yml

```diff
@@ -1,5 +1,5 @@
 name: Go Build
-on: [push]
```

Comment:
This is actually changed on `master`, but I realized that PRs coming from user forks don't trigger a build/test.